doc/develop/toolchains/zephyr_sdk.rst

```diff
@@ -20,32 +23,39 @@ The Zephyr SDK supports the following target architectures:
 * x86 (32-bit and 64-bit)
 * Xtensa
 
+.. _toolchain_zephyr_sdk_bundle_variables:
+
+Installation bundle and variables
+*********************************
+
 The Zephyr SDK bundle supports all major operating systems (Linux, macOS and
 Windows) and is delivered as a compressed file.
 The installation consists of extracting the file and running the included setup
 script. Additional OS-specific instructions are described in the sections below.
 
-By default, the Zephyr build system assumes that the toolchain will be obtained
-from the Zephyr SDK. You can enforce this by setting the environment variable
-:envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to ``zephyr``.
-
-If you install the Zephyr SDK outside any of the default locations (listed in
-the operating system specific instructions below), you must register the Zephyr
-SDK in the CMake package registry by running the setup script or setting
-:envvar:`ZEPHYR_SDK_INSTALL_DIR` to point to the Zephyr SDK installation
-directory.
-
-You can also set :envvar:`ZEPHYR_SDK_INSTALL_DIR` to point to a directory
-containing multiple Zephyr SDKs, allowing for automatic toolchain selection. For
-example, you can set ``ZEPHYR_SDK_INSTALL_DIR`` to ``/company/tools``, where the
-``company/tools`` folder contains the following subfolders:
-
-* ``/company/tools/zephyr-sdk-0.13.2``
-* ``/company/tools/zephyr-sdk-a.b.c``
-* ``/company/tools/zephyr-sdk-x.y.z``
-
-This allows the Zephyr build system to choose the correct version of the SDK,
-while allowing multiple Zephyr SDKs to be grouped together at a specific path.
+Toolchain variant variable
+   By default, the Zephyr build system assumes that the toolchain will be obtained
+   from the Zephyr SDK. You can enforce this by setting the environment variable
+   :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` to ``zephyr``.
+
+SDK installation directory variable
```

Comment:
I do not see how this is an improvement over the existing documentation.

It would indeed be useful to have an "environment variable glossary" page under `doc/develop/toolchains` that lists all the toolchain-related environment variables (referenced by `:envvar:`) with a detailed description, but what is done here seems like a half-baked solution.